### PR TITLE
[Getting Started] mesheryctl command to create and set kubernetes context

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR makes it so that kubernetes users can have a functional meshconfig with Kubernetes context.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: _includes/getting-started.html [hunk 1]
```diff
@@ -88,10 +88,12 @@ <h3>Kubernetes User:</h3>
             <!-- Kubernetes - mesheryctl -->
             <h4 style="font-size: 1.5em;">Using mesheryctl</h4>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system start</code></p>
+              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system context create k8s -p kubernetes -s
+                mesheryctl system start</code></p>
```
Comment (alphaX86) [inline _includes/getting-started.html:92]: image

IG the text box can be of big size.... Let me try....

Comment (alphaX86) [reply _includes/getting-started.html:92]: @leecalcote I'll try opening a PR to the same branch via my fork coz it's not accepting the origin on mine

Comment (alphaX86) [reply _includes/getting-started.html:92]: Seems I can't edit the changes here in mine.... I'll add suggested changes then

Comment (alphaX86) [reply _includes/getting-started.html:92]: Set height to 70px and then we're good to go!

**Review by alphaX86 — COMMENTED** [review]
This to max the text box

File: _includes/getting-started.html (hunk 1 above)
Comment (leecalcote) [reply _includes/getting-started.html:92]: @alphaX86 thanks much for this. Great catch.

By the way, if you ever feel like suggesting the change via CTRL+G (or CMD+G) in your reviews, please don't hesitate. It'll be good to collaboratively editing a PR with you.

File: _includes/getting-started.html [hunk 2]
```diff
@@ -88,10 +88,12 @@ <h3>Kubernetes User:</h3>
             <!-- Kubernetes - mesheryctl -->
             <h4 style="font-size: 1.5em;">Using mesheryctl</h4>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system start</code></p>
+              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system context create k8s -p kubernetes -s
```
Comment (leecalcote) [inline _includes/getting-started.html:91]:
```suggestion
              <p class="code-box" style="height: 70px; line-height: 30px;"><code style="position: absolute;">mesheryctl system context create k8s -p kubernetes -s
```

**Pull request merged**